tools/docker/docker-compose.yml

```diff
@@ -14,12 +14,27 @@ services:
             MYSQL_DATABASE: ragnarok
             MYSQL_USER: ragnarok
             MYSQL_PASSWORD: ragnarok
-    server:
+
+    builder:
+        image: "rathena:local"
+        container_name: "builder"
+        volumes:
+            - "../..:/rathena" # mount git repo directory inside container
+            - "./asset/inter_conf.txt:/rathena/conf/import/inter_conf.txt" # load db connection
+            - "./asset/char_conf.txt:/rathena/conf/import/char_conf.txt"   #localdev login-char relation
+            - "./asset/map_conf.txt:/rathena/conf/import/map_conf.txt"     #localdev char-map relation
+        init: true # helps with signal forwarding and process reaping
+        tty: true
+        stdin_open: true
+        build:
+            context: .
+            dockerfile: Dockerfile
+    
+    login:
         image: "rathena:local"
-        container_name: "rathena"
+        container_name: "rathena-login"
+        command: "-c ./login-server"
```

Comment:
these should probably be using the absolute path just for clarity's sake
`command: "-c /rathena/login-server"`